pkg/cmd/taskrun/list.go

```diff
@@ -202,22 +208,33 @@ func list(p cli.Params, task string, limit int, labelselector string) (*v1alpha1
 	return trs, nil
 }
 
-func printFormatted(s *cli.Stream, trs *v1alpha1.TaskRunList, c clockwork.Clock) error {
+func printFormatted(s *cli.Stream, trs *v1alpha1.TaskRunList, c clockwork.Clock, allnamespaces bool) error {
 	if len(trs.Items) == 0 {
 		fmt.Fprintln(s.Err, emptyMsg)
 		return nil
 	}
 
 	w := tabwriter.NewWriter(s.Out, 0, 5, 3, ' ', tabwriter.TabIndent)
-	fmt.Fprintln(w, "NAME\tSTARTED\tDURATION\tSTATUS\t")
+	indexString := "NAME\tSTARTED\tDURATION\tSTATUS\t"
+	if allnamespaces {
+		indexString = indexString + "NAMESPACE\t"
+	}
+	fmt.Fprintln(w, indexString)
 	for _, tr := range trs.Items {
-
-		fmt.Fprintf(w, "%s\t%s\t%s\t%s\t\n",
+		trNamespace := ""
+		formatString := "%s\t%s\t%s\t%s\t"
```

Comment:
Same comment as pipelineruns:

The `formatString` is not lining up the namespace values correctly:

```
NAME                                                STARTED          DURATION     STATUS               NAMESPACE   
sleep1s-run-qxqfs                                   21 seconds ago   4 seconds    Succeeded   test        
sleep1s-run-m79sf                                   59 seconds ago   12 seconds   Succeeded   test        

```

Reply:
I am not able to reproduce what you are seeing there.
```
NAME                           STARTED   DURATION   STATUS   NAMESPACE
gcs-resource-2t859             ---       ---        ---      jenkins-operator-test
gcs-resource-gst97             ---       ---        ---      tekton-pipelines
task-to-list-files-run-v4crv   ---       ---        ---      tekton-pipelines
```
It seems more of an issue with `indexString` as the distance between `STATUS` and `NAMESPACE` seems to be more than 1 tabspace.

Reply:
Same as pipelineruns.
``` 
NAME                                                STARTED      DURATION     STATUS                            NAMESPACE      
vote-api-buildah-18zk1t-deploy-gktrl                6 days ago   20 seconds   Failed                   test-local-2   
vote-api-buildah-18zk1t-build-v5z9m                 6 days ago   1 minute     Succeeded                test-local-2   
```

Reply:
Updated : Using templates instead of string theory